workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Injection/WorldCommandRequestSender.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Improbable.Gdk.Core.Commands;
+using Improbable.Worker;
+using Unity.Entities;
+using UnityEngine.Assertions.Must;
+using Entity = Unity.Entities.Entity;
+
+#region Diagnostic control
+
+// ReSharper disable UnusedMember.Local
+
+#endregion
+
+namespace Improbable.Gdk.Core.GameObjectRepresentation
+{
+    public static partial class WorldCommandsRequirables
+    {
+        [InjectableId(InjectableType.WorldCommandRequestSender, InjectableId.NullComponentId)]
+        [InjectionCondition(InjectionCondition.RequireNothing)]
+        public class WorldCommandRequestSender : IInjectable
+        {
+            private readonly Entity entity;
+            private readonly EntityManager entityManager;
+
+            private WorldCommandRequestSender(Entity entity, EntityManager entityManager)
+            {
+                this.entity = entity;
+                this.entityManager = entityManager;
+            }
+
+            public WorldCommands.ReserveEntityIds.Request ReserveEntityIds(
+                uint numberOfEntityIds,
+                object context = null)
```

Comment:
We probably do also want the timeout to be exposed. Probably with a default value.

Reply:
yes, will be added as part of createrequest fix